change/@fluentui-react-tabs-00427f46-b6ea-45d8-989f-a7e2d94bd0bf.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Tab component skips animation for falsy values",
```

Comment:
not sure this description fits the fix rather the origin issue. can we please improve this for changelog  ?

good practice is also to explicitly specify semver type: fix/feat/chore etc

ty